src/SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests/Steps/VerifyIdentityAndRegistrationSteps.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using AutoFixture;
+using FluentAssertions;
+using Newtonsoft.Json;
+using SFA.DAS.ApprenticeCommitments.Api.ErrorHandler;
+using SFA.DAS.ApprenticeCommitments.Apis.InnerApi;
+using SFA.DAS.ApprenticeCommitments.Application.Commands.VerifyIdentityRegistration;
+using TechTalk.SpecFlow;
+using WireMock.RequestBuilders;
+using WireMock.ResponseBuilders;
+
+namespace SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests.Steps
+{
+    [Binding]
+    [Scope(Feature = "VerifyIdentityAndRegistration")]
+    public class VerifyIdentityAndRegistrationSteps
+    {
+        private readonly TestContext _context;
+        private VerifyIdentityRegistrationCommand _command;
+        private Fixture _f;
+        private string _validEmail;
+        private string _validNI;
+        private Guid _registrationId;
+        private Guid _userIdentityId;
+        private ErrorItem _errorItem;
+
+        public VerifyIdentityAndRegistrationSteps(TestContext context)
+        {
+            _context = context;
+            _f = new Fixture();
+            _validEmail = "a@a.com";
```

Comment:
AutoFixture can create email addresses too ✉️ 
```suggestion
            _validEmail = _f.Create<MailAddress>().Address;
```

(It requires `using System.Net.Mail;`)

Reply:
nice to know thx